.prettierignore

```diff
@@ -1,8 +1,38 @@
 # Ignore artifacts:
-build
-coverage
-.next
 .lighthouseci
+# Dependencies
+node_modules
+.pnpm-store
+
+# Build outputs
+.next
 out
-storybook-static
-.github
\ No newline at end of file
+dist
+build
+
+# Testing
+.stryker-tmp
+**/.stryker-tmp/**
+coverage
+.nyc_output
+**/*.snap
+playwright-report/
+lhci-reports-desktop/
+lhci-reports-mobile/
+
+# Storybook
+storybook-static/
+
+# Cache
+.cache
+.eslintcache
+.stylelintcache
+.npm
+
```

Comment:
Please remove unnecessary comments and be consistent with the amount of new lines between sections, sometimes there is one new line and sometimes there is two lines between sections